datafusion/core/src/dataframe.rs

```diff
@@ -520,7 +520,13 @@ impl DataFrame {
         self.plan.schema()
     }
 
-    /// Return the logical plan represented by this DataFrame.
+    /// Return the raw (i.e. unoptimized) logical plan represented by this DataFrame.
+    pub fn to_raw_logical_plan(&self) -> LogicalPlan {
```

Comment:
I wonder if a better name would be `to_unoptimized_plan`?

Reply:
Sure. Do we need to specify that it is a logical as opposed to physical plan in the name?

Reply:
I don't think so, because the return type is `LogicalPlan`

Reply:
Sure!

Reply:
@andygrove  Done!